.coveragerc

```diff
@@ -594,6 +594,9 @@ omit =
     homeassistant/components/mastodon/notify.py
     homeassistant/components/matrix/*
     homeassistant/components/mcp23017/*
+    homeassistant/components/meater/__init__.py
+    homeassistant/components/meater/const.py
+    homeassistant/components/meater/sensor.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.
```suggestion
```

Reply:
Let's do that in a follow-up